src/v0/util/index.test.js

```diff
@@ -1163,3 +1163,203 @@ describe('getBodyFromV2SpecPayload', () => {
     expect(utilities.getBodyFromV2SpecPayload(input)).toEqual([1, 2, 3]);
   });
 });
+
+describe('isNewStatusCodesAccepted', () => {
```

Comment:
@utsabc @koladilip what's use-case around FeatureFlagMiddleware ? do we have any doc for this ? I'm not much aware of when this got introduced

Reply:
This is introduced when @chandumlg was there for proxy v0 to v1 and tradedesk gzip support